package.json

```diff
@@ -58,12 +58,12 @@
   },
   "dependencies": {
```

Comment:
@dudiq
1. When I click on "Record selfie video", I got permission denied error immediately. Nothing recording obviously. No asks about access to camera, etc, also camera is enabled.
![image](https://user-images.githubusercontent.com/13561530/84951592-535a6600-b0f9-11ea-98eb-fcaa587c835c.png)
2. No mp4 icons available. Update please package.json in vault
3. Some files cannot be uploaded, breaks with error on 12-20%. Maybe kodek. Just FYI

Reply:
you need [this](https://github.com/endpass/endpass-connect/pull/182) pr from connect for resolve error in console

Reply:
about mp4 icons, this repo does not depend from vault

Reply:
When I have selfie recorded, I can't upload. "The uploaded file is broken or have unknown format". Seems it's server error, so @kazurus 

@dudiq please change color of "continue" button to white, according to the design.
![image](https://user-images.githubusercontent.com/13561530/85001120-f0e68180-b15b-11ea-8830-9348afd58e40.png)

Reply:
fixed button

Reply:
@dudiq maybe make this buttons disabled?
![image](https://user-images.githubusercontent.com/13561530/85127391-016c2a00-b238-11ea-8b77-f6872b66d7bd.png)